openmmtools/integrators.py

```diff
@@ -1453,6 +1453,11 @@ def add_integrator_steps(self, splitting, measure_shadow_work, measure_heat, ORV
         """
         Override the base class to insert reset steps around the integrator.
         """
+
+        # Integrate
```

Comment:
Calling `self.addUpdateContextState()` here is not appropriate because you call `super(NonequilibriumLangevinIntegrator, self).add_integrator_steps()` a few lines later, which also calls `self.addUpdateContextState()`, which would mean you call this twice per step.